### PR TITLE
add timezone aware conversion function for ephem dates

### PR DESCRIPTION
**Pull request by 1kastner**

### CODE REVIEW

**Comment by 1kastner** [comment]
It is a bit cumbersome because I develop on a windows machine and the `time.tzset` is not available. Further many other tests break without obvious reasons (except issues discussed in #115)

File: ephem/__init__.py [hunk 1]
```diff
@@ -536,14 +536,28 @@ def next_pass(self, body, singlepass=True):
...
+
+
+def localtime(date):
+    """Convert a PyEphem date into naive local time, returning a Python datetime."""
+    seconds, microseconds = _convert_to_seconds_and_microseconds(date)
     y, m, d, H, M, S, wday, yday, isdst = _localtime(seconds)
     return _datetime(y, m, d, H, M, S, microseconds)
 
+
+def timezoned_time(date, tzinfo):
```
Comment (brandon-rhodes) [inline ephem/__init__.py]: Could we name this `to_timezone()` instead?

**Comment by 1kastner** [comment]
@brandon-rhodes does it meet your expectations now?

**Comment by brandon-rhodes** [comment]
Hah! I just realized: I really prefer `from datetime import …` in Python programs, because I find `datetime.datetime` (or its equivalent) so redundant, but I was trying to decide whether to even mention it here, and so of course I couldn't make up my mind immediately and never finished my review.

So, I'll ask for the `import` style I already had to be restored, if you're willing, and you can just add `timedelta` as the other thing you import from it.

(I'm going on vacation starting this morning, and plan to look at this pull request again right when I return; so if you don't see immediate action from me, that's why!)

**Comment by 1kastner** [comment]
Ok, all right. Then I will import everything with its name preceded by a low dash. Enjoy your holidays!

**Comment by 1kastner** [comment]
Btw, do you have a date in mind when this could be published? You already said this is more like maintenance and less an active development, but still this could be helpful for some people?

**Comment by brandon-rhodes** [comment]
I don't have an exact date in mind, no; I'll have to return to the Appveyor build and remember whether it's easy or difficult to get a release out for Windows. I wonder if I was thorough enough to leave myself a release checklist sitting around; if not then I'll need to put one together.

**Comment by brandon-rhodes** [comment]
And it's merged! We should add this to the docs soon if we want this to be discoverable for folks. Are you interested in adding it at least to the Quick Reference?

**Comment by 1kastner** [comment]
Wow, thank you very much! I can have a look at how it could be placed next to http://rhodesmill.org/pyephem/quick#local-time

**Comment by brandon-rhodes** [comment]
Thanks, I look forward to another pull request! :)

**Pull request merged**